docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,136 @@
+# Release 469 (14 Jan 2025)
+
+## General
+
+* Add geospatial constructor {func}`ST_GeomFromKML`. ({issue}`24297`)
+* Refrain from join pushdown for modified tables. ({issue}`24447`)
+* Allow deactivating the spooling protocol with the `spooling_protocol_enabled`
+  session property. ({issue}`24655`)
+* Fix coordinator out of memory issues caused by a Jetty upgrade in Trino 468. ({issue}`24572`) 
```

Comment:
Not sure if we refer to version of regression and cause typically .. happy to remove or reword. Thoughts @wendigo @martint ?

Reply:
It's both coordinator and worker. I'd just say "improve stability of the cluster under load" ?